src/Sylius/Bundle/ShippingBundle/tests/Validator/ValidDeliveryTimeRangeValidatorTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Bundle\ShippingBundle\Validator;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\ShippingBundle\Validator\Constraint\ValidDeliveryTimeRange;
+use Sylius\Bundle\ShippingBundle\Validator\ValidDeliveryTimeRangeValidator;
+use Sylius\Component\Shipping\Model\ShippingMethodInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+use Symfony\Component\Validator\Violation\ConstraintViolationBuilderInterface;
+
+final class ValidDeliveryTimeRangeValidatorTest extends TestCase
```

Comment:
```suggestion
final class ValidDeliveryTimeRangeValidatorTest extends \Symfony\Component\Validator\Test\ConstraintValidatorTestCase
```
Symfony provides a dedicated abstract class for testing validators.